style.css

```diff
@@ -1,3 +1,88 @@
 .hide {
   display: none;
 }
+
+p:nth-of-type(2) {
+  background: red;
+}
+
+.full-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  background-color: aliceblue;
+}
```

Comment:
- [ ] Your application should match this [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png)
![Screenshot from 2021-10-21 17-40-31](https://user-images.githubusercontent.com/40503704/138321217-84de60fd-7887-40ea-aa14-7d14827a42a2.png)


Here is what your app look like
![Screenshot from 2021-10-21 17-42-34](https://user-images.githubusercontent.com/40503704/138321159-fc8da0a0-4005-4273-8f78-f06649d83f85.png)
- No background on each book item
- No divider between the book list and the new book form